### PR TITLE
Initial code and tests

### PR DESCRIPTION
**Pull request by bas080**
Tests are not working somehow. In future fix and add more tests

### CODE REVIEW

File: test/setup.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// import 'aurelia-polyfills';
+// import {initialize} from 'aurelia-pal-browser';
+// initialize();
```
Comment (VMBindraban) [inline test/setup.js:3]: This file seems a bit pointless.

File: src/gravatar.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    if (defaults.indexOf(this.defaultTo) === -1) {
+      throw new Error(`${this.defaultTo} is not a valid gravatar default to image. Choose one of the following: ` + JSON.stringify(defaults));
+    }
+
+    return this.defaultTo;
+  }
+
+  @computedFrom('size', 'email', 'defaultGravatar', 'forceDefault', 'secure')
+  get url() {
+    let base;
```
Comment (VMBindraban) [inline src/gravatar.js:32]: You mixed up the secure url and this is nicer.

``` js
 let base = 'http://www.gravatar.com/avatar';
 if (this.secure) {
   base = 'https://secure.gravatar.com/avatar';
 }
```

File: doc/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+### 1.0.0-beta.1.1.0 (2016-03-23)
```
Comment (VMBindraban) [inline doc/CHANGELOG.md:1]: This seems to be the changelog of the aurelia skeleton-plugin.

File: test/setup.js (hunk 1 above)
Comment (RWOverdijk) [inline test/setup.js:3]: The point of these is to initialize the test browser (for karma). When set up properly, this is useful.

File: src/gravatar.js (hunk 2 above)
Comment (RWOverdijk) [inline src/gravatar.js:32]:
``` js
let base = (this.secure ? 'https://secure' : 'http://www') + '.gravatar.com/avatar';
```

File: test/setup.js (hunk 1 above)
Comment (VMBindraban) [inline test/setup.js:3]: I was referring that all the lines are commented out.

**Comment by doktordirk** [comment]
Re setup. Probably not needed atm. generally, 
- Add polyfill there, if -well- polyfils are needed (should't be needed for Evergreen aka Firefox i think).
- Add aurelia-pal-browser+initialize, if the src/test uses aurelia-pal

**Comment by RWOverdijk** [comment]
@bas080 When you're done with the view manager, perhaps take a look at the feedback here :)

**Pull request merged**